graylog2-server/src/main/resources/swagger/index.html.template

```diff
@@ -67,6 +72,13 @@
 </head>
 
 <body>
+<div class="warning-box" style="visibility:hidden;">
```

Comment:
We should consider moving this to a JSX file so we can use our component library

Reply:
Sorry, ignore this, thought it was a react template file, not swagger, probably the best way to accomplish this